src/pages/suivi/index.tsx

```diff
@@ -27,16 +46,51 @@ export default function Suivi() {
             sont exemptés d’autorisation préalable.
           </Text>
           <p className="mb-3">
-            Nous n’utilisons aucun outil pour surveiller les usages du site.
+            Nous utilisons pour cela{" "}
+            <ExternalLink
+              href="https://matomo.org/"
+              className="underline decoration-1 hover:decoration-2 transition ease-in-out delay-50 duration-300 underline-offset-4"
+              title="Matomo"
+            >
+              Matomo
+            </ExternalLink>
+            , un outil{" "}
+            <ExternalLink
+              href="https://matomo.org/free-software/"
+              className="underline decoration-1 hover:decoration-2 transition ease-in-out delay-50 duration-300 underline-offset-4"
+              title="Matomo libre"
+            >
+              libre
+            </ExternalLink>
+            , paramétré pour être en conformité avec la{" "}
+            <ExternalLink
+              href="https://www.cnil.fr/fr/cookies-et-autres-traceurs/regles/cookies-solutions-pour-les-outils-de-mesure-daudience"
+              className="underline decoration-1 hover:decoration-2 transition ease-in-out delay-50 duration-300 underline-offset-4"
+              title="recommandation « Cookies »"
+            >
+              recommandation « Cookies »
+            </ExternalLink>{" "}
```

Comment:
@manuhabitela please note this `ExternalLink` will need to be refactored when rebasing my work with your recent works on title. That's why I didn't spend much time on it.

Also I would like to introduce a wrapper, to have a unique component for external link in legal pages.

Reply:
Might be, let's see after your PR

Reply:
You can remove the `title` props that are not necessary anymore.